### PR TITLE
Symfony5

### PR DESCRIPTION
**Pull request by michellesanver**
| Q | A
| --- | ---
| Branch? | 2.0
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Tests pass? | yes/no
| Fixed tickets | #1241
| License | MIT

Upgrading to Symfony 5 and adding php 7.4 support in travis while we're at it

### CODE REVIEW

**Comment by michellesanver** [comment]
Currently blocked by: https://github.com/php-enqueue/enqueue-dev/pull/987

**Comment by rogoit** [comment]
Would be good if we have it. Thx for your work!!

File: composer.json [hunk 1]
```diff
@@ -20,14 +20,14 @@
...
-        "symfony/templating": "^3.4|^4.2",
-        "symfony/translation": "^3.4|^4.2"
+        "symfony/asset": "^3.4|^4.2|^5.0",
+        "symfony/filesystem": "^3.4|^4.2|^5.0",
+        "symfony/finder": "^3.4|^4.2|^5.0",
+        "symfony/framework-bundle": "^3.4|^4.2|^5.0",
+        "symfony/options-resolver": "^3.4|^4.2|^5.0",
+        "symfony/process": "^3.4|^4.2|^5.0",
+        "symfony/templating": "^3.4|^4.2|^5.0",
+        "symfony/translation": "^3.4|^4.2|^5.0"
```
Comment (franmomu) [inline composer.json]: mmmm I think the bundle doesn't use any of `"symfony/translation"` and `"symfony/asset"`

**Comment by conradfr** [comment]
php-enqueue/enqueue-dev#987 has been merged.

**Comment by dbu** [comment]
we need to allow enqueue `0.10` to profit from this. enqueue seems to still be in beta state with 0.x releases - for 0.x semver considers every minor version potentially BC breaking...

**Comment by conradfr** [comment]
Sure, although it doesn't seem they changed much https://github.com/php-enqueue/enqueue-dev/compare/0.9.15...0.10.0

**Comment by dbu** [comment]
> Sure, although it doesn't seem they changed much php-enqueue/enqueue-dev@0.9.15...0.10.0

yeah, then i am hopeful we can simply allow that version and things will be fine. (i wanted to explain why we need to explicitly allow the version, did not want to imply that it would be a problem)

**Comment by lsmith77** [comment]
enqueue-bundle dep needs to allow also `^0.10`

https://github.com/liip/LiipImagineBundle/blob/f8bf61e75a605ca0ac7bdf34727905eacda8fca8/composer.json#L38

**Comment by weaverryan** [comment]
Hi! And happy holidays to everyone!

I wanted to help push this forward (and I selfishly need this for a screencast!) so I took Michelle's work (as well as work from a different PR) and polished everything in #1256. That should now be ready.

Thank you!

**Comment by michellesanver** [comment]
Solved by #1256

**Comment by michellesanver** [comment]
I see test coverage officially fell but I am happy with our test coverage and merging despite that.

**Pull request merged**